azure/packages/external-controller/webpack.config.js

```diff
@@ -40,6 +41,9 @@ module.exports = (env) => {
 				libraryTarget: "umd",
 			},
 			plugins: [
+				new webpack.DefinePlugin({
+					"process.env.FLUID_CLIENT": JSON.stringify(fluidClient),
+				}),
 				new webpack.ProvidePlugin({
```

Comment:
Id be curious if we are correctly handling env generally here, but lets unblock azure run at this point.